src/SprykerMiddleware/Zed/Process/ProcessDependencyProvider.php

```diff
@@ -0,0 +1,369 @@
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addEncodingService(Container $container): Container
+    {
+        $container[static::SERVICE_UTIL_ENCODING] = function (Container $container) {
+            return new ProcessToUtilEncodingServiceBridge($container->getLocator()->utilEncoding()->service());
+        };
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addProcessService(Container $container): Container
+    {
+        $container[static::SERVICE_PROCESS] = function (Container $container) {
+            return $container->getLocator()->process()->service();
+        };
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addLogHandlers($container): Container
+    {
+        $container[static::MIDDLEWARE_LOG_HANDLERS] = function () {
+            return $this->getLogHandlers();
+        };
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addLogProcessors($container): Container
+    {
+        $container[static::MIDDLEWARE_LOG_PROCESSORS] = function () {
+            return $this->getLogProcessors();
+        };
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addDefaultLoggerConfigPlugin($container): Container
+    {
+        $container[static::MIDDLEWARE_DEFAULT_LOG_CONFIG_PLUGIN] = function () {
+            return $this->getDefaultLoggerConfigPlugin();
+        };
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addGenericTranslatorFunctions($container): Container
+    {
+        $container[static::MIDDLEWARE_GENERIC_TRANSLATOR_FUNCTIONS] = function () {
+            return $this->getGenericTranslatorFunctionsStack();
+        };
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addGenericValidators($container): Container
+    {
+        $container[static::MIDDLEWARE_GENERIC_VALIDATORS] = function () {
+            return $this->getGenericValidatorsStack();
+        };
+
+        return $container;
+    }
+
+    /**
+     * @return \SprykerMiddleware\Zed\Process\Dependency\Plugin\Log\MiddlewareLoggerConfigPluginInterface
+     */
+    protected function getDefaultLoggerConfigPlugin(): MiddlewareLoggerConfigPluginInterface
```

Comment:
Will you have not-default logger plugin? Will they be available at the same time?
Please, no defaults here.